packages/magic/src/validation.ts

```diff
@@ -0,0 +1,20 @@
+import Joi from 'joi'
+import type { MagicInitOptions } from './types'
+
+const magicOptions = Joi.object({
+  apiKey: Joi.string().required(),
+  userEmail: Joi.string()
```

Comment:
```suggestion
  userEmail: Joi.string().allow(null)
```
We could allow null here to make is easy for initialization since localStorage returns null if the value is not there. So the magic module could be initialized directly like this:

```javascript
const magic = magicModule({ apiKey: 'dsdfash3r43hf', userEmail: localStorage.getItem('magicUserEmail') })
```